src/platform/plugins/shared/unified_histogram/public/chart/hooks/use_time_range.test.tsx

```diff
@@ -87,17 +87,7 @@ describe('useTimeRange', () => {
     );
     expect(result.current.timeRangeDisplay).toMatchInlineSnapshot(`
       <EuiText
-        css={
-          Object {
-            "map": undefined,
-            "name": "1vgo99t",
-            "next": undefined,
-            "styles": "
-          padding: 0 8px 0 8px;
-        ",
-            "toString": [Function],
-          }
-        }
+        css="unknown styles"
```

Comment:
and I think I found the cure for these regressions https://emotion.sh/docs/@emotion/babel-plugin